tctl-snap/tctl-plugins/cmd/store.go

```diff
@@ -0,0 +1,15 @@
+package cmd
+
+import (
+	"os"
+	"path/filepath"
+)
+
+// TokenStoreLocation returns the location of the token store given a service
+// name following the pattern of ~/.local/share/serviceName/tokens.
+func TokenStoreLocation(serviceName string) string {
+	if p := os.Getenv("XDG_DATA_HOME"); p != "" {
```

Comment:
Any chance these environment variables would be missing? I imagine any user exotic enough to not have these would be able to debug any issues themselves anyway. Just checking if we have any edge cases that we need to cover here.